src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -119,12 +266,12 @@ void Snippet::initSupportedPrimitiveDescriptors() {
 
     const size_t ndims = outputShapes[0].getRank();
     // Domain sensitive operations support only Planar layout
-    const bool isOnlyPlanarApplicable = snippet->has_domain_sensitive_ops();
+    const bool isOnlyPlanarApplicable = snippetAttrs.snippet->has_domain_sensitive_ops();
     const bool isChannelsFirstApplicable = dnnl::impl::utils::one_of(ndims, 1u, 2u, 3u, 4u, 5u) && dimRanksAreEqual && !isOnlyPlanarApplicable;
     // Todo: Snippets currently don't support per-channel broadcasting of Blocked descriptors because
     //  canonicalization can't distinguish between <N, C, H, W, c> and <N, C, D, H, W> cases.
     //  See snippets::op::Subgraph::canonicalize for details.
-    bool isBlockedApplicable = dnnl::impl::utils::one_of(ndims,  4u, 5u) && dimRanksAreEqual && !isOnlyPlanarApplicable;
+    bool isBlockedApplicable = dnnl::impl::utils::one_of(ndims,  3u, 4u, 5u) && dimRanksAreEqual && !isOnlyPlanarApplicable;
```

Comment:
Do we have the tests for blocked 3D layout? If not, could you please add? Thanks!

Reply:
Yes, we have some 3D block dynamic eltwise tests cases. After enable dynamic node, it's tokenized and run as subgraph, that's why I extend here. I can add some specific snippets 3d block test case. Thanks

Reply:
There are already 3d block dynamic test, such as in https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/subgraph_tests/src/eltwise_caching.cpp#L531. If snippets test disable eltwise dynamic test, I will add separate cases. What do you think? Thanks Alexandra!

Reply:
Now 3d eltwise test have snippets backend. Thanks.